base/Hero.js

```diff
@@ -61,6 +64,13 @@ export class Hero extends ControllableChar {
         this.trying_to_push = false;
         this.trying_to_push_direction = null;
         this.push_timer = null;
+        this.field_abilities_list = {};
```

Comment:
Field abilities would be the next thing that I would refactor. In my plans, a NPC could also cast field abilities. I think that field abilities classes can continue to be separate classes. When casting, the field ability instance should receive a ControllableChar instance (if NPC, no need for PP check and other MainChars related attributes checks).

In the future, all that initialized data in the game load will have a better place (I didn't think anything about it yet).